src/plugins/platforms/freerdp/qfreerdppeer.cpp

```diff
@@ -1521,12 +1521,12 @@ bool QFreeRdpPeer::setPointer(const POINTER_LARGE_UPDATE *largePointer, Qt::Curs
 		lpointer->cacheIndex = cacheIndex;
 		lpointer->width = largePointer->width;
 		lpointer->height = largePointer->height;
-#if FREERDP_VERSION_MAJOR < 3
-		lpointer->xPos = largePointer->hotSpotX;
-		lpointer->yPos = largePointer->hotSpotY;
-#else
+#if FREERDP_VERSION_MAJOR == 3 && FREERDP_VERSION_MINOR >= 5 || FREERDP_VERSION_MAJOR > 3
```

Comment:
I'd suggest lowering the check on `FREERDP_VERSION_MINOR` to 1 because the renaming of `hotSpot[X|Y]` is way older than 3.5 (early 3.0 IIRC)

Reply:
My bad, I was apparently off by one year when I checked the git blame for those features /o\